Spec/RealtimeClientChannel.swift

```diff
@@ -70,11 +67,11 @@ class RealtimeClientChannel: QuickSpec {
                     }
```

Comment:
Also, the `subscribe` on line 52 should be moved above the `channel2.attach()`, to make sure it happens before it is attached.